project/MultiNode.scala

```diff
@@ -68,6 +68,8 @@ object MultiNode extends AutoPlugin {
     SbtMultiJvm.multiJvmSettings ++
       inConfig(MultiJvm)(scalafmtConfigSettings) ++
       Seq(
+        // Hack because 'provided' dependencies by default are not picked up by the multi-jvm plugin:
+        managedClasspath in MultiJvm ++= (managedClasspath in Compile).value.filter(_.data.name.contains("silencer-lib")),
```

Comment:
so it fails at compile time? at runtime it shouldn't be needed

Reply:
Correct. Very strange.